_posts/2025-04-17-tls-wizard-for-wildfly.adoc

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: 'Introduction to a GUI wizard to configure TLS in WildFly'
+date: 2025-04-17
+tags: elytron gui
+synopsis: Learn how to use a GUI wizard to configure one-way TLS in WildFly.
+author: theashiot
+link: https://wildfly-security.github.io/wildfly-elytron/blog/_posts/gui-tls-wizard.adoc/
```

Comment:
@theashiot , I tried to access the link but got a 404, the correct ink seems:https://wildfly-security.github.io/wildfly-elytron/blog/gui-tls-wizard/